compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -337,6 +337,8 @@ LogicalResult isAtBoundary(Operation *op) {
 
 void addGPUTileAndFusePassPipeline(OpPassManager &funcPassManager,
                                    const GPUPipelineOptions &pipelineOptions) {
+  tileAndDistributeToWorkgroup(funcPassManager, /*useForall=*/true,
+                               /*convertToDpsOptions=*/std::nullopt);
   if (pipelineOptions.useIgemmConvolution) {
     funcPassManager.addPass(createConvolutionToIGEMMPass());
   }
```

Comment:
This should go after `createConvolutionToIGEMMPass`. The distribution tile sizes set on convs for igemm are actually in the iteration space of the resulting matmul after the igemm transformation.